Packs/Azure/Integrations/Azure/Azure.py

```diff
@@ -715,6 +764,252 @@ def storage_blob_containers_create_update_request(
+
+        demisto.debug(f"Request headers: {self.headers}")
+
+    def get_storage_container_properties_request(self, account_name: str, container_name: str) -> requests.Response:
+        """
+        Retrieve properties for the specified Container.
+
+        Args:
+            container_name (str): Container name.
+
+        Returns:
+            Response: API response from Azure.
+
+        """
+        params = assign_params(restype="container")
+        full_url = f"https://{account_name}.blob.core.windows.net/{container_name}"
+        self.storage_container_set_headers()
+
+        response = self.http_request(method="GET", full_url=full_url, params=params, resp_type="response")
+
+        return response  # type: ignore[return-value]
+
+    def create_storage_container_request(self, container_name: str, account_name: str) -> requests.Response:
+        """
+        Create a new Container under the specified account.
+
+        Args:
+            container_name (str): Container name.
+            account_name (str): Storage account name.
+
+        Returns:
+            Response: API response from Azure.
+
+        """
+        params = assign_params(restype="container")
+        full_url = f"https://{account_name}.blob.core.windows.net/{container_name}"
+        self.storage_container_set_headers()
+
+        response = self.http_request(method="PUT", full_url=full_url, params=params, resp_type="response")
+
+        return response  # type: ignore[return-value]
+
+    def delete_storage_container_request(self, container_name: str, account_name: str) -> None:
+        """
+        Delete Container under the specified account.
+
+        Args:
+            container_name (str): Container name.
+            account_name (str): Storage account name.
+
+        Returns:
+            Response: API response from Azure.
+
+        """
+        params = assign_params(restype="container")
+        full_url = f"https://{account_name}.blob.core.windows.net/{container_name}"
+        self.storage_container_set_headers()
+
+        self.http_request(method="DELETE", full_url=full_url, params=params, resp_type="response")
+
+    def storage_container_create_blob_request(
+        self, container_name: str, account_name: str, file_entry_id: str, file_name: str = ""
+    ) -> requests.Response | dict[str, Any]:  # noqa: E501
+        """
+        Create or update Blob under the specified Container.
+
+        Args:
+            container_name (str): Container name.
+            file_entry_id (str): File War room Entry ID.
+            file_name (str): File name. Default is XSOAR file name.
+
+        Returns:
+            Response: API response from Azure.
+
+        """
+
+        xsoar_file_data = demisto.getFilePath(file_entry_id)  # Retrieve XSOAR system file path and name, given file entry ID.
+        xsoar_system_file_path = xsoar_file_data["path"]
+        blob_name = file_name if file_name else xsoar_file_data["name"]
```

Comment:
Move this 3 code lines to storage_container_blob_create_command, and call storage_container_create_blob_request with the correct file/blob name.
This way the readable output will also have the correct name.